lib/constants/providers.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:juejin/exports.dart';
+
+final authProvider = FutureProvider<UserAuthen>((ref) {
+  // TODO
```

Comment:
We should prefer `flutter_style_todos` in the repo (maybe I'll add it later). For example:
```dart
// TODO(shirne): Construct the model correctly.
```